torch/_inductor/scheduler.py

```diff
@@ -1772,11 +1769,6 @@ def check_all_pairs(nodes):
                 check_all_pairs(node_grouping)
 
         possible_fusions.sort(key=self.score_fusion_key, reverse=True)
-        if fusion_log.isEnabledFor(logging.DEBUG):
```

Comment:
I feel it's still useful to keep them for debugging purpose.

Reply:
I think it's redundant since the nodes that don't get fused will get logged anyway in `speedup_by_fusion`. Every possible fusion will already have at least one log line printed out, either to say we are fusing them, or why we are not fusing them.

Reply:
Is it possible that some fusions are not logged because priorities are low? i.e. High-pri fusions are already done so low-pri fusions are ignored.

Reply:
I don't think so... `can_fuse` and `will_fusion_create_cycle` also print out a log if they determine a fusion is not possible due to changes in the graph from higher-pri fusions.